packages/admin-next/dashboard/src/components/table/data-table/data-table-filter/date-filter.tsx

```diff
@@ -118,123 +119,136 @@ export const DateFilter = ({
 
   return (
     <Popover.Root modal open={open} onOpenChange={handleOpenChange}>
-      <DateDisplay label={label} value={displayValue} onRemove={handleRemove} />
-      <Popover.Portal>
-        <Popover.Content
-          data-name="date_filter_content"
-          align="start"
-          sideOffset={8}
-          collisionPadding={24}
-          className={clx(
-            "bg-ui-bg-base text-ui-fg-base shadow-elevation-flyout h-full max-h-[var(--radix-popper-available-height)] w-[300px] overflow-auto rounded-lg"
-          )}
-          onInteractOutside={(e) => {
-            if (e.target instanceof HTMLElement) {
-              if (
-                e.target.attributes.getNamedItem("data-name")?.value ===
-                "filters_menu_content"
-              ) {
-                e.preventDefault()
+      <DateDisplay
+        label={label}
+        value={displayValue}
+        onRemove={handleRemove}
+        readonly={readonly}
+      />
+      {!readonly && (
```

Comment:
These are just indentation changes, no actual code changed other than the readonly check.